### PR TITLE
Update Set-TransportRule.md

### PR DESCRIPTION
**Pull request by m-nestorovic**
Clarified what is required in order for message to be considered as InOrganization.

Work Item ID in the ADO: 3490530

### CODE REVIEW

**Comment by chrisda** [comment]
Sorry, man. I thought I merged your changes, and I had a correction to make, so I did it in another PR, which is now conflicting with your PR.

So, closing your PR.

**Comment by m-nestorovic** [comment]
No problem at all. I see that changes are applied so everything is good.

Thanks!

File: exchange/exchange-ps/exchange/Set-TransportRule.md [hunk 1]
```diff
@@ -2115,7 +2115,7 @@ In on-premises Exchange, this exception is available on Mailbox servers and Edge
 
 The ExceptIfFromScope parameter specifies an exception that looks for the location of message senders. Valid values are:
 
-- InOrganization: The sender is a mailbox, mail user, group, or mail-enabled public folder in your organization or The sender's email address is in an accepted domain that's configured as an authoritative domain or an internal relay domain, and the message was sent or received over an authenticated connection.
+- InOrganization: The message was sent or received over an authenticated connection **AND** the sender is a mailbox, mail user, group, or mail-enabled public folder in your organization, **OR** the sender's email address is in an accepted domain that's configured as an authoritative domain or an internal relay domain in your organization.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-TransportRule.md:2118]:
```suggestion
- InOrganization: The message was sent or received over an authenticated connection *and* the sender is a mailbox, mail user, group, or mail-enabled public folder in your organization, *or* the sender's email address is in an accepted domain that's configured as an authoritative domain or an internal relay domain in your organization.
```

File: exchange/exchange-ps/exchange/Set-TransportRule.md [hunk 2]
```diff
@@ -3175,7 +3175,7 @@ In on-premises Exchange, this condition is available on Mailbox servers and Edge
 
 The FromScope parameter specifies a condition that looks for the location of message senders. Valid values are:
 
-- InOrganization: The sender is a mailbox, mail user, group, or mail-enabled public folder in your organization or The sender's email address is in an accepted domain that's configured as an authoritative domain or an internal relay domain, and the message was sent or received over an authenticated connection.
+- InOrganization: The message was sent or received over an authenticated connection **AND** the sender is a mailbox, mail user, group, or mail-enabled public folder in your organization, **OR** the sender's email address is in an accepted domain that's configured as an authoritative domain or an internal relay domain in your organization.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-TransportRule.md:3178]:
```suggestion
- InOrganization: The message was sent or received over an authenticated connection *and* the sender is a mailbox, mail user, group, or mail-enabled public folder in your organization, *or* the sender's email address is in an accepted domain that's configured as an authoritative domain or an internal relay domain in your organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @m-nestorovic, kindly commit the suggested changes. For reference, we do not use capitalization and bold type for emphasis. Many thanks.

**Pull request closed without merging**